.travis.yml

```diff
@@ -43,7 +43,8 @@ install:
   - pip install -U pip setuptools
   - pip install docutils==$DOCUTILS
   - pip install -r test-reqs.txt
+  - if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then python3.6 -m pip install mypy-lang 'typed-ast<1.0'; fi
```

Comment:
I'm pinning `typed-ast` to `<1.0`, because current `typed-ast` conflicts with `mypy-lang` unfortunately ...
```
$ pip install --upgrade typed-ast
Collecting typed-ast
  Using cached typed_ast-1.0.1-cp36-cp36m-macosx_10_11_x86_64.whl
Installing collected packages: typed-ast
  Found existing installation: typed-ast 0.6.3
    Uninstalling typed-ast-0.6.3:
      Successfully uninstalled typed-ast-0.6.3
Successfully installed typed-ast-1.0.1
$ mypy sphinx
You must install the typed_ast package before you can run mypy with `--fast-parser`.
You can do this with `python3 -m pip install typed-ast`.
```